commands/define.js

```diff
@@ -41,39 +107,38 @@ exports.run = (client, message, args) => {
           // Only run when data is recieved with statusCode of 200
           if (!error && response.statusCode == 200) {
             // Parse JSON data
-            const data = JSON.parse(body)
+            const data = JSON.parse(body);
             // Abstract text
-            const abstract = data['AbstractText']
+            const abstract = data['AbstractText'];
             // If no abstract text is found then:
             if (!abstract) {
-              let msg = `Cannot get exact definition, what are you looking for?\n\n`
-              const RelatedTopics = data['RelatedTopics']
+              let msg = `Definition not provided, please check these following links or try \`!define wiki ${searchText}\` for wikipedia results.\n\n`;
+              const RelatedTopics = data['RelatedTopics'];
               RelatedTopics.every((topic, index) => {
                 /**
-                * For some related topics, no links are provided,
-                * hence those are not collected and the loop is
-                * exited.
-                */
+                 * For some related topics, no links are provided,
+                 * hence those are not collected and the loop is
+                 * exited.
+                 */
                 if (!topic['Text']) {
-                  return false
+                  return false;
                 } else {
-                  msg += `${topic['FirstURL']}\n`
-                  msg += `*${topic['Text']}*\n\n`
-                  return true
+                  msg += `${topic['FirstURL']}\n`;
+                  msg += `*${topic['Text']}*\n\n`;
+                  return true;
                 }
-              })
-              message.channel.send(msg)
-            } else { // If abstract text is found: 
-                let msg = `Definition for __${args[0]}__`
-                msg += '```'
-                msg += abstract
-                msg += '```'
-                message.channel.send(msg)
+              });
+              message.channel.send(msg);
+            } else {
+              // If abstract text is found:
+              let msg = `Definition for \`${searchText}\` `;
+              msg += '```';
```

Comment:
Can you not combine these into one line using `\n` as a stand-in for new lines?

Reply:
We can do this in one line and don't have have to use `\n` at all, because code blocks are `display:block`. so it will go something like this:
```js
let msg = `Definition for \`${searchText}\` \`\`\` `;
```
Now using three back ticks in a separate lines make it look better and easy to understand.

Reply:
Alright no problem, just wanted to see your reasoning. I'll merge this as is.